sdk/src/lib.ts

```diff
@@ -0,0 +1,114 @@
+import type { Project, OpenOptions, EmbedOptions } from './interfaces';
+import type { VM } from './VM';
+import { Connection, getConnection } from './connection';
+import { openNewProject, createProjectFrameHTML } from './generate';
+import {
+  replaceAndEmbed,
+  buildProjectQuery,
+  elementFromElementOrId,
+  openTarget,
+  getOrigin,
+} from './helpers';
+
+/**
+ * Get a VM instance for an existing StackBlitz project iframe.
+ */
+export function connect(frameEl: HTMLIFrameElement): Promise<VM> {
+  if (!frameEl || !frameEl.contentWindow) {
+    return Promise.reject('Provided element is not an iframe.');
+  }
+  const connection = getConnection(frameEl) || new Connection(frameEl);
+  return connection.pending;
+}
+
+/**
+ * Open an existing StackBlitz project in a new tab (or in the current window).
+ */
+export function openProject(project: Project, options?: OpenOptions) {
+  openNewProject(project, options);
+}
+
+/**
+ * Open an existing StackBlitz project in a new tab (or in the current window).
+ */
+export function openProjectId(projectId: string, options?: OpenOptions) {
```

Comment:
We can't change this because it's an existing part of our public API:
https://developer.stackblitz.com/docs/platform/javascript-sdk/#sdkopenprojectidprojectid-opts

Same for all the functions exported from `sdk/src/lib.ts`.